tests/test_knowledge_panels.py

```diff
@@ -91,3 +93,36 @@ def test_hunger_game_kp_label_with_value():
             ]
         }
     }
+
+
+class MockResponse:
+    def json(self):
+        return {
+            "data-quality": {
+                "elements": [
+                    {"element_type": "text", "total_issues": 0, "text_element_1": {}}
+                ],
+                "source_url": "https://world.openfoodfacts.org/country/Hungary/packaging/plastic-box/data-quality",
+                "description": "This is a data-quality issues related to packaging based for Hungary",
+            }
+        }
```

Comment:
This part is really to "mimick" the Open Food Facts application, not your application (this is a classic mistake when people mock for the first time :-)). Remember you use because you don't want to depend of open food facts application.

So this really should return a piece of json taken from, eg.  https://fr-en.openfoodfacts.org/label/bio/data-quality.json